src/R/Editor/Impl/Outline/ROutliningTagger.cs

```diff
@@ -4,14 +4,15 @@
 using System;
 using Microsoft.Languages.Editor.Outline;
 using Microsoft.Languages.Editor.Services;
+using Microsoft.Languages.Editor.Shell;
 using Microsoft.R.Editor.Document;
 using Microsoft.VisualStudio.Text.Tagging;
 
 namespace Microsoft.R.Editor.Outline {
     internal sealed class ROutliningTagger : OutliningTagger {
         [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Reliability", "CA2000:Dispose objects before losing scope")]
         public ROutliningTagger(REditorDocument document)
-            : base(document.EditorTree.TextBuffer, new ROutlineRegionBuilder(document)) {
+            : base(document.EditorTree.TextBuffer, new ROutlineRegionBuilder(document, EditorShell.Current)) {
```

Comment:
What can be imported? `ROutlineRegionBuilder`?

---
In reply to: [68843223](https://github.com/Microsoft/RTVS/pull/2022#discussion_r68843223) [](ancestors = 68843223)